Archive/Imo/Imo2021Q1.lean

```diff
@@ -99,28 +98,22 @@ theorem exists_finset_3_le_card_with_pairs_summing_to_squares (n : ℕ) (hn : 10
   · simp only [Finset.mem_insert, Finset.mem_singleton]
     rintro d (rfl | rfl | rfl) <;> constructor <;> linarith only [hna, hab, hbc, hcn]
 
-end Imo2021Q1
```

Comment:
This was correct. Otherwise, the main result will be named `Imo2021Q11.imo2021_q1`.

Reply:
Yep, I'll revert this change.